New hosts UI features (#1736)
theforeman/foreman-documentation: guides/common/modules/proc_adding-a-host-to-host-collections.adoc

[id="adding-a-host-to-host-collections_{context}"]
= Adding a host to Host collections

Host can be added to a host collection from the {ProjectWebUI}.

.Procedure
. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
. Click the name of the hosts you want to modify.

> . Click the *Add to host collection* button under the *Host collections* card.

Reviewer: ```suggestion
. In the *Host collections* card, click the vertical ellipsis and select *Add host to collections*.
```

. Select the host collection.
. Click *Add*.

.Verification

> * The *Host collections* card under the *Overview* tab now shows the host collection your host belongs to.

Reviewer 2: Is the "Verification" section new? cc @Lennonka @adamlazik1

Author: @maximiliankolb It's part of the standard CCS template. It's been there for quite some time. The section is used in other files in this repo too guides/common/modules/proc_enabling-capsule-in-UI.adoc:

Reviewer 2: I am not against this at all. It's just that I haven't seen this before. Thanks for explaining.

What is CCS? Are the templates public?
